docs/config/ping-federate-example.md

```diff
@@ -91,10 +91,10 @@ MIICnzCCAYcCBgFkqEAQCDANBgkqhkiG9w0BAQsFADATMREwDwYDVQQDDAhhbGZyZXNjbzA
 2. Navigate to **SP Connections** and select **Create New**.
 3. Ensure the **Browser SSO Profiles** connection template with *Protocol SAML 2.0* is selected and then click **Next**.
 4. On the **Connection Options** tab ensure only **Browser SSO** is selected.
-5. On the **General Info** tab 
+>5. On the **General Info** tab 
   * Add the *Redirect URI* value that you noted in step >>> but remove */broker/saml/endpoint*.
   * Add a connection name.
-  * 
+  * Add the *Redirect URI* value that you noted in step >>> under *Base URL* but this time remove everything up to the first forward slash such as */auth/realms/alfresco/broker/saml/endpoint*.
```

Comment:
I take it that some of these marks placeholders?

Reply:
Sorry this is very messy because I just committed what I had before I left. All occurrences of ">>>" were just me reminding myself to go back and add numbers to those points. The value that I was talking about is the highlighted value here: 
![image](https://user-images.githubusercontent.com/42536841/59701647-fa0ddd00-91ed-11e9-831a-99e0914f8a11.png)